README.md

```diff
@@ -2,7 +2,32 @@
 
 A gem to document components in GOV.UK frontend applications.
 
-## Usage
+![Screenshot of component guide](docs/screenshot.png)
+
+## Write components in your application
+
+Components are packages of template, style, behaviour and documentation that live in your application.
```

Comment:
Package is fine I think, and terser than 'associated group of files'. If the component files aren't grouped well enough then the problem isn't a naming one.